src/main/java/com/fredsonchaves07/moviecatchapi/domain/useCases/user/CreateUserUseCase.java

```diff
@@ -73,22 +69,11 @@ private void saveUser() {
 
     private void sendMail() {
         String token = getToken();
-        String urlConfirmation = getUrlTokenConfirmation(token);
-        String content = """
-                Olá! Tudo bem?
-                Para confirmar seu cadastro por favor clique no link abaixo
-                                
-                """ + urlConfirmation;
-        String subject = "Bem vindo ao moviecatch";
-        sendEmailService.send(new MessageEmailDTO(subject, user.getEmail(), content));
+        String subject = "Welcome to MovieCatch! \uD83D\uDE00";
+        sendEmailService.send(new MessageEmailDTO(subject, user.getEmail(), token), "welcome_mail");
```

Comment:
As a template is being used create a list of variables to be passed as optional parameters to the template